content/en/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace.md

```diff
@@ -2,14 +2,18 @@
 title: Configure Minimum and Maximum CPU Constraints for a Namespace
 content_type: task
 weight: 40
+description: >-
+  Define a range of valid CPU resource limits for a namespace, so that every new Pod
+  in that namespace falls within the range you configured.
 ---
 
 
 <!-- overview -->
 
-This page shows how to set minimum and maximum values for the CPU resources used by Containers
-and Pods in a namespace. You specify minimum and maximum CPU values in a
-[LimitRange](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#limitrange-v1-core)
+This page shows how to set minimum and maximum values for the CPU resources used by containers
```

Comment:
General comment about capitalization. Not sure why "Pod" is capitalized throughout, but "containers" was changed to lower case here. My inclination would be to not capitalize either, unless one is referring to literal values. But Pod is consistently capitalized here.

Reply:
We could lowercase pod in some places. When we're talking about things with `kind: Pod`, we usually write it in title case.

Reply:
https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects

Reply:
There's no specific guidance about whether container is an API object.